homeassistant/components/netgear_lte/__init__.py

```diff
@@ -137,62 +148,83 @@ def get_modem_data(self, config):
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up Netgear LTE component."""
-    if DOMAIN not in hass.data:
-        websession = async_create_clientsession(
-            hass, cookie_jar=aiohttp.CookieJar(unsafe=True)
+    hass.data[DATA_HASS_CONFIG] = config
+
+    if lte_config := config.get(DOMAIN):
+        for entry in lte_config:
+            hass.async_create_task(
+                hass.config_entries.flow.async_init(
```

Comment:
If the import flow abort and fails we should tell the user that in the repair issue. Probably await the import flow here and use different translation keys depending on the result and abort reason of the flow.

Reply:
Awaiting the flow result causes startup to hang, ideas?